plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -1,34 +1,94 @@
 /*
-  	Copyright 2020 Google LLC
+ 	Copyright 2020 Google LLC
 
-  	Licensed under the Apache License, Version 2.0 (the "License");
-  	you may not use this file except in compliance with the License.
-  	You may obtain a copy of the License at
+ 	Licensed under the Apache License, Version 2.0 (the "License");
+ 	you may not use this file except in compliance with the License.
+ 	You may obtain a copy of the License at
 
-  		https://www.apache.org/licenses/LICENSE-2.0
+ 		https://www.apache.org/licenses/LICENSE-2.0
 
-  	Unless required by applicable law or agreed to in writing, software
-  	distributed under the License is distributed on an "AS IS" BASIS,
-  	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  	See the License for the specific language governing permissions and
-  	limitations under the License.
- */
+ 	Unless required by applicable law or agreed to in writing, software
+ 	distributed under the License is distributed on an "AS IS" BASIS,
+ 	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ 	See the License for the specific language governing permissions and
+ 	limitations under the License.
+*/
 
 package com.google.play.core.godot.assetpacks;
 
 import androidx.annotation.NonNull;
 
 import org.godotengine.godot.Godot;
 import org.godotengine.godot.plugin.GodotPlugin;
+import org.godotengine.godot.plugin.SignalInfo;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.HashSet;
 
+/**
+ * This class is served as a middleware, to expose the Play Core Java library to the Godot runtime,
+ * so Godot Play Asset Delivery front-end API is able to access the related Java APIs. This class is
+ * intended to be compiled to an .AAR Android Library, which can be loaded into Godot Engine as a
+ * Godot Android Plugin.
+ */
 public class PlayAssetDelivery extends GodotPlugin {
-    public PlayAssetDelivery(Godot godot) {
-        super(godot);
-    }
-
-    @NonNull
-    @Override
-    public String getPluginName() {
-        throw new UnsupportedOperationException("Method not implemented yet!");
-    }
+
+  public PlayAssetDelivery(Godot godot) {
+    super(godot);
+  }
+
+  @NonNull
+  @Override
+  public String getPluginName() {
+    return "PlayAssetDelivery";
+  }
+
+  /** Returns a list of all the available plugin method names that the Godot runtime can call. */
```

Comment:
This formatting is automatically apply by the google-java-formatter intellij plugin